azurerm/resource_arm_notification_hub_namespace.go

```diff
@@ -40,25 +40,39 @@ func resourceArmNotificationHubNamespace() *schema.Resource {
 			"location": azure.SchemaLocation(),
 
 			"sku": {
-				Type:     schema.TypeList,
-				Required: true,
-				MaxItems: 1,
+				Type:          schema.TypeList,
+				Optional:      true,
+				Deprecated:    "This property has been deprecated in favour of the 'sku_name' property and will be removed in version 2.0 of the provider",
+				ConflictsWith: []string{"sku_name"},
+				MaxItems:      1,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"name": {
-							Type:     schema.TypeString,
-							Required: true,
-							ForceNew: true,
+							Type:             schema.TypeString,
+							Required:         true,
+							ForceNew:         true,
 							ValidateFunc: validation.StringInSlice([]string{
 								string(notificationhubs.Basic),
 								string(notificationhubs.Free),
 								string(notificationhubs.Standard),
-							}, false),
+							}, true),
```

Comment:
We shouldn't be changing this?
```suggestion
							}, false),
```

Reply:
As before, shouldn't this be
```suggestion
							}, false),
```